src/cmd/root.go

```diff
@@ -11,6 +11,9 @@ import (
 
 var cfgFile string
 
+var historyFilePathConfigKey = "historyFile"
```

Comment:
Should those be constants?
https://golang.org/doc/effective_go.html#constants

Reply:
https://github.com/tome-sh/tome/pull/1/commits/06b34d6420392eb67ef36cf57f71c91d42bf978a